front-end/src/components/settings-view/SettingsView.js

```diff
@@ -1,15 +1,30 @@
 import React, { Component } from 'react';
+import { changeLanguage } from 'actions';
+import { connect } from 'react-redux';
 
 class SettingsView extends Component {
     
-
     render() {
+        let {i18n} = this.props;
         return (
             <div>
-             Settings are here
+                <p>Choose language:</p>
+                <button onClick = {() => this.props.onClick('en')}>{i18n.enButton}</button>
```

Comment:
beware of onClick effect on touch screen.
I suggest using react newest pointer event handler

Reply:
https://reactjs.org/blog/2018/05/23/react-v-16-4.html